app/assets/javascripts/validation.js

```diff
@@ -10,69 +10,228 @@ $(document).ready(function () {
         minlength: 8
       },
       password2: {
-        equalTo: "#password1"
+        equalTo: '#password1'
       }
     },
     messages: {
       password1: {
-        required: "Please enter password",
-        minlength: "Password should be minimum 8 characters long"
+        required: 'Please enter password',
+        minlength: 'Password should be minimum 8 characters long'
       },
       email: {
-        required: "Please enter email",
-        email: "Invalid email address"
+        required: 'Please enter email',
```

Comment:
Is changing double quotes to single quotes a necessity here?
Just checking

Reply:
No, it's not, it's just a personal preference(editor defaults).

Reply:
Kindly don't do changes which are not required. It can cause trouble at later phases. I hope you understand my concern

Reply:
Yes, I understand, should I switch back double quotes? Thanks.